recipes/turbo_turtle/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "turbo_turtle" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl-aea/turbo-turtle/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e89f218c8d95484f3b0b8b61427c0bd82779226301436c991535966bac4d140d
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  entry_points:
+    - turbo-turtle = turbo_turtle._main:main
+  ignore_run_exports:
+    - python_abi
```

Comment:
Why are you doing this?

Reply:
https://github.com/conda/conda-build/issues/3956#issuecomment-845711269

Reply:
Unless behavior has changed in conda-build, the package will inherit an unnecessary Python ABI dependency when building against the conda-forge channel.

I'll try running without it to see if the runtime dependencies still look correct.

Reply:
looks like the behavior has changed. I've removed these lines.